src/layouts/CyrillicLayout.js

```diff
@@ -1,5 +1,9 @@
-export default [
-	['й', 'ц', 'у', 'к', 'е', 'н', 'г', 'ш', 'щ', 'з', 'х', 'ъ'],
-	['ф', 'ы', 'в', 'а', 'п', 'р', 'о', 'л', 'д', 'ж', 'э'],
-	['я', 'ч', 'с', 'м', 'и', 'т', 'ь', 'б', 'ю'],
-];
+export default {
```

Comment:
Can you add layouts to the export in the main file?  https://github.com/WiaczeslawP/react-screen-keyboard/blob/master/Keyboard.js